core/themes/seven/css/style.css

```diff
@@ -1496,19 +1496,22 @@ table.update {
 }
 
 /* Maintenance page styles */
-body.maintenance-page {
+body.maintenance-page,
+body.user-login-page {
   background: #E8E6E5;
   padding: 1em;
 }
-body.maintenance-page #page {
+body.maintenance-page #page,
+.user-login-form {
```

Comment:
I'm getting confused here whether this is a body class or not. It seems to be inconsistent whether `.user-login-form` or `body.user-login-form` is used. Is there a reason for that? I would just prefix all of them with `body` if that's what intended here.